src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -76,7 +79,7 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
 
-        Person personToEdit = lastShownList.get(index.getZeroBased());
+        Person personToEdit = lastShownList.get(index.getZeroBased()); // get the person we are editting
```

Comment:
There seems to be a spelling error here. Instead of `editting`, it should be `editing`.

Reply:
got it.